x-pack/plugin/ilm/src/main/java/org/elasticsearch/xpack/ilm/action/TransportStopILMAction.java

```diff
@@ -50,7 +51,8 @@ protected AcknowledgedResponse read(StreamInput in) throws IOException {
     @Override
     protected void masterOperation(Task task, StopILMRequest request, ClusterState state, ActionListener<AcknowledgedResponse> listener) {
         clusterService.submitStateUpdateTask("ilm_operation_mode_update",
-                new AckedClusterStateUpdateTask<AcknowledgedResponse>(request, listener) {
+                new AckedClusterStateUpdateTask<AcknowledgedResponse>(Priority.URGENT, request, listener) {
```

Comment:
This also needs to change the cluster state update on line 366 of IndexLifecycleService (`submitOperationModeUpdate`), that is the second half that moves the state from `STOPPING` -> `STOPPED`

Reply:
Ah yes, good shout